package.json

```diff
@@ -210,6 +210,7 @@
     "react-router-dom": "^5.2.0"
   },
   "optionalDependencies": {
-    "@folio/plugin-find-user": "^6.0.0"
+    "@folio/plugin-find-user": "^6.0.0",
+    "@folio/plugin-find-instance": "*"
```

Comment:
Hello @zburke. Could you please clarify, should we change here plugin version after our changes for it will be released? Thanks!

Reply:
IMO `*` is never an acceptable dependency. I would expect a `^6` dep would protect you here, and allow your application to receive the newest-available version, even if it's not released yet, since I anticipate only minor/patch releases for that plugin in Lotus.

Reply:
Done